lib/logstash/inputs/file.rb

```diff
@@ -146,8 +146,9 @@ class LogStash::Inputs::File < LogStash::Inputs::Base
   # When the file input discovers a file that was last modified
   # before the specified timespan in seconds, the file is ignored.
   # After it's discovery, if an ignored file is modified it is no
-  # longer ignored and any new data is read. The default is 24 hours.
-  config :ignore_older, :validate => :number, :default => 24 * 60 * 60
+  # longer ignored and any new data is read. By default, this option is
```

Comment:
Maybe add a comment that the value is in seconds? removing the comment about the default value removes the only documentation about the unit of `ignore_older`